src/dotnet/Program.cs

```diff
@@ -183,7 +183,7 @@ private static void PrintInfo()
             Reporter.Output.WriteLine();
             Reporter.Output.WriteLine("Product Information:");
             Reporter.Output.WriteLine($" Version:     {Product.Version}");
```

Comment:
Should we add more whitespace before `{Product.Version}` so the values stay lined up vertically?